styles/global.css

```diff
@@ -2,6 +2,7 @@ html {
   font-size: var(--font-size, 15px);
   width: 100%;
   width: 100vw;
+  font-family: var(--font-family, 'FiraGO');
```

Comment:
@sxzz I'm not fully sure how these things works, but I think this is already a prepare for https://github.com/elk-zone/elk/issues/1179

I assume then in a follow-up PR we can dynamically set the `--font-family` variable and swap it